src/common/utils/formatting.test.ts

```diff
@@ -0,0 +1,25 @@
+import { getAriaLabel } from "./formatting";
+
+describe('getAriaLabel', () => {
+  it("should return 'key' and 'data' values in aria label", () => {
+
+    const datapoint = {
+      key: 'JSON',
+      data: 25
+    };
+    const result = getAriaLabel(datapoint);
+    expect(result).toEqual('JSON: 25');
+  });
+
+  it("should return 'x' and 'y' values in aria label", () => {
+
+    const datapoint = {
+      x: 'Windows -> Win7',
```

Comment:
Do we want it to be `->`?

Reply:
This is for nested data like the TreeMap. Do we want to have it something like "Win7 nested into Windows" (there could be multiple levels of nesting though)?
<img width="477" alt="Screenshot 2023-12-29 at 1 15 27 AM" src="https://github.com/reaviz/reaviz/assets/33908100/9ff00fa0-9d26-430e-9fe6-bd70be816026">

Reply:
OK - I'm not sure. You tell us what the interwebs says for this.

Reply:
Ok, I replaced it with an arrow `→`.
So now the screenreader reads it as "Windows right arrow Win7" which I think is better.